tracer/src/Datadog.Trace/Vendors/MessagePack/Formatters/CollectionFormatter.cs

```diff
@@ -8,7 +8,7 @@
 using System.Collections.ObjectModel;
 using System.Linq;
 
-#if NETSTANDARD || NETFRAMEWORK
+#if NETSTANDARD || NETFRAMEWORK || NETCOREAPP
```

Comment:
Um, what does that leave?! 😕 And what were they using instead on .NET Core 🤔

Reply:
It looks like they only had netstandard and netfx build targets, so I'm not sure why they didn't just remove the ifdef entirely: https://github.com/neuecc/MessagePack-CSharp/blob/v1.9.11/src/MessagePack/MessagePack.csproj#L4

Reply:
The support other platforms, like Unity.

Reply:
FWIW, this is how we fixed it before (#912). It's less code changes from the original vendor code (compared to removing the `#if`).

Current code in `master`:
https://github.com/DataDog/dd-trace-dotnet/blob/b44aa4bee2de8f8ee50b6557b4c00c107395bc0b/tracer/src/Datadog.Trace/Vendors/MessagePack/Formatters/CollectionFormatter.cs#L11